docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -53,89 +53,13 @@ and generate credentials that the plugin will use for authentication.
 Handler plugin on your Universal Forwarder host:
 
-<Tabs dropdownCaption="Version" dropdownSelected="Latest Release">
-  <TabItem options="Latest Release" label="Download the Binary">
-
-  On the host where you are running the Universal Forwarder, execute the
-  following commands:
-
-  ```code
-  $ curl -L -O https://get.gravitational.com/teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-  ```
-
-  Move the `teleport-event-handler` binary to `/usr/local/bin`.
-  </TabItem>
-
-  <TabItem options="Latest Release" label="Pull via Docker">
-
-  On the host where you are running the Universal Forwarder, execute the
-  following command:
-
-  ```code
-  $ docker pull public.ecr.aws/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
-  ```
-
-  </TabItem>
-
-  <TabItem options="From Source" label="Build via Docker">
-  You must have Docker installed on your Universal Forwarder host. 
-
-  Log in to the host and execute the following commands to build the plugin:
-
-  ```code
-  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
-  $ cd teleport-plugins/event-handler/build.assets
-  $ make build
-  ```
-
-  You can find the compiled binary within your clone of the `teleport-plugins`
-  repo, with the file path, `event-handler/build/teleport-event-handler`. Move
-  this to `/usr/local/bin`.
-
-  </TabItem>
-  <TabItem options="From Source" label="Build via Go">
-
-  You will need Go >= (=teleport.golang=) installed.
-
-  Run the following commands on your Universal Forwarder host:
-
-  ```code 
-  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
-  $ cd teleport-plugins/event-handler
-  $ go build
-  ```
-
-  The resulting executable will have the name `event-handler`. To follow the
-  rest of this guide, rename this file to `teleport-event-handler` and move it
-  to `/usr/local/bin`.
-
-  </TabItem>
-</Tabs>
+(!docs/pages/includes/install-event-handler.mdx!)
 
 ### Generate a starter config file
 
-On the host where you are running the Universal Forwarder and the Teleport Event
-Handler plugin, execute the following command to generate a sample configuration
-that we will edit later:
+Generate a configuration file with placeholder values for the Teleport Event Handler plugin. Later in this guide, we will edit the configuration file for your environment.
 
-```code
-$ teleport-event-handler configure . <Var name="teleport-cluster-name"/>
-```
-
-The `teleport-event-handler configure` command generates several files that you
-will refer to when generating credentials for the Teleport Event Handler plugin
-and configuring the Universal Forwarder:
-
-|File(s)|Purpose|
-|---|---|
-| `ca.crt` | Self-signed CA certificate that the Teleport Event Handler plugin and the Universal Forwarder will use to validate one another's TLS certificates.|
```

Comment:
This table includes some Splunk-specific context that is lost in the new partial

Reply:
Resolved in b097a42ae92d40695296cd3b47603563832ef0f5